lib/server.js

```diff
@@ -84,6 +84,11 @@ function createServer (options, httpHandler) {
 
         multipleBindings.call(this, server, httpHandler, options, listenOptions, () => {
           this[kState].listening = true
+
+          for (const hook of this[kHooks].onListen) {
```

Comment:
Why dont you use hookRunnerApplication?

Reply:
Good question. We will investigate hookRunnerApplication.